daemon/daemon.go

```diff
@@ -208,12 +210,12 @@ func main() {
 			first = false
 			//clean start require wifi AP down so we can get SSIDs
 			cw.Disable()
-			//TODO only wait if wlan0 is managed
-			//wait time period (TBD) on first run to allow wifi connections
-			time.Sleep(40000 * time.Millisecond)
 			//remove previous state flag, if any on deamon startup
 			utils.RemoveFlagFile(waitFlagPath)
 			utils.RemoveFlagFile(manualFlagPath)
```

Comment:
sounds to me that before starting the loop these same two lines removing any previous flag file can now be removed. Btw, have you tested nothing is broken with this solution. I thought about this same fix but I was afraid of introducing regressions